src/OSSupport/Errors.cpp

```diff
@@ -5,13 +5,13 @@
 
 AString GetOSErrorString( int a_ErrNo)
 {
-	char buffer[ 1024 ];
 	AString Out;
 
 	#ifdef _WIN32
 
+	wchar_t buffer[1024];
 	FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM, nullptr, a_ErrNo, 0, buffer, ARRAYCOUNT(buffer), nullptr);
-	Printf(Out, "%d: %s", a_ErrNo, buffer);
+	Printf(Out, "%d: %S", a_ErrNo, buffer);
```

Comment:
Well, I've installed the Chinese (Simplified) language pack on me computer, and have founds that in short, getting Unicode to print on a console window is for me impossible, since writing to the console window apparently isn't truly done in Unicode but is instead dependant on the system locale.

The GUI part of things (even MessageBox called from a console app) fully supports Unicode, but alas, we're not a graphical application.

Of course, tests show that the ANSI version doesn't work with Chinese, displaying all '?'s in the IDE, even when the Wide version produced a sentence correctly.

Since there doesn't appear to be any way to fix this, I'll revert the Wide changes to this file, since that's more correct in context of the rest of the server.